### DISCUSSION TITLE
Unable to search on body of documents, only Title

### DISCUSSION THREAD

**Discussion by markatwork8823** [microsoft-search/pnp-modern-search#2639] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm unclear if this feature currently exists but as I can't seem to find it in the documentation or the past issues, I will raise it. 

**Describe the solution you'd like**
I have a document Library and have built a Search Page using the Modern PnP Web Parts. We have a business requirement to search both the Document Title and the Body of a document however currently after searching for "Test", I see ["Title="test"] and it does not show documents that have "Test" in the document body but not the Title. I cannot find any configuration options to set this so I'm unsure if this is a missing feature or I have not configured my Search Results Web Part correctly. 

**Describe alternatives you've considered**
I have attempted to configure the Query Template in both the Web Part Properties and also configure a new Result Source showing Title:{SearchTerms} OR Body:{SearchTerms}. This appears to work from testing directly from the Result Source but the expected results do not show when searching using the PnP Web Parts which is configured to use the Custom Result Source. Entering the same query or similar queries in the Web Part Properties has failed to work. 

Appreciate your help in clarifying how to configure full text search of documents. 

Thanks

**Additional context**

**Comment by wobba** [comment, marked as answer]
By default both title and contents of a file is searches, so no need to have property queries. Or is your requirement to search only Title+Body and not any other metadata/fields?

Also property query using `=` means equals. Use `:` and it's contains like `Title:(test)`.

Take a look at the docs at https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference#grouping-property-restrictions-within-a-kql-query

**Comment by markatwork8823** [comment]
Thanks @wobba, I'm not sure what happened in my configuration but it's just searching the title. I've been able to get it to work starting with a new page and re-configuring from scratch. Thanks for your help
